includes/settings.php

```diff
@@ -217,6 +219,33 @@ function license_key_callback() {
 	<?php
 }
 
+/**
+ * Output media handling options.
+ *
+ * @since 1.2.2
+ */
+function media_handling_callback() {
+	$settings = Utils\get_settings();
+	?>
+
+	<ul class="media-handling">
+		<li>
+			<input <?php checked( $settings['media_handling'], 'featured' ); ?> type="radio" value="featured" name="dt_settings[media_handling]">
+			<span class="description">
```

Comment:
I would not wrap these in a span with the `.description` class, as it makes the text italic. The class is really meant for further explanatory text for text entry fields. They do need to be `label`s though. The label can be wrapped around the `input` so you don't have to add an `id`.